data/json/items/armor/bespoke_armor/custom_boots.json

```diff
@@ -61,36 +61,24 @@
     "armor": [ { "encumbrance": 24, "coverage": 100, "covers": [ "foot_l", "foot_r" ] } ]
   },
   {
-    "id": "boots_xlsurvivor",
+    "id": "xl_boots_survivor",
     "type": "ARMOR",
-    "category": "armor",
     "name": { "str": "pair of XL survivor boots", "str_pl": "pairs of XL survivor boots" },
     "description": "A massive pair of customized leather boots armored with Kevlar.  Balancing comfort and protection, these boots will keep you alive even when knee-deep in the dead.",
     "weight": "2460 g",
     "volume": "5 L",
-    "price": 24000,
-    "price_postapoc": 4000,
-    "to_hit": -1,
-    "bashing": 1,
-    "material": [ "kevlar", "leather" ],
     "symbol": "[",
-    "looks_like": "boots_combat",
-    "color": "brown",
-    "warmth": 15,
     "material_thickness": 4,
```

Comment:
This kind a make sense, just like in `xl_armguard_acidchitin`, but up to debate.